src/Yoti.Auth/YotiClientEngine.cs

```diff
@@ -244,14 +256,25 @@ private static void HandleOtherAttributes(YotiUserProfile profile, AttrpubapiV1.
             }
         }
 
-        private string GetEndpoint(string token, string nonce, string timestamp, string sdkId)
+        private string GetEndpoint(HttpMethod httpMethod, string path, string token, string nonce, string timestamp, string sdkId, byte[] payload)
         {
-            return string.Format("/profile/{0}?nonce={1}&timestamp={2}&appId={3}", token, nonce, timestamp, sdkId);
+            string endpoint = string.Format("/{0}/{1}?nonce={2}&timestamp={3}&appId={4}", path, token, nonce, timestamp, sdkId);
+            if (payload != null && payload.Length > 0)
+            {
+                string urlSafePayloadString = Conversion.BytesToUrlSafeBase64(payload);
+                endpoint += string.Format("&payload={0}", urlSafePayloadString);
```

Comment:
please remove "&payload", otherwise it will not work. It should be just "&" + payload_value

Reply:
Ok, I did it in a similar way to Moussa did for PHP, so does he have to change the PHP one too? - https://github.com/getyoti/yoti-php-sdk/pull/18/files#diff-f97f8bdde6e955330461dafd9d7be9fcR122

Reply:
Correct! Thanks for spotting this!

Reply:
I've sent Moussa a message so he is aware of the necessary change